<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statementScreen</class>
 <widget class="QMainWindow" name="statementScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>statement</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(127, 127, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="account_lable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 204, 255);</string>
    </property>
    <property name="text">
     <string>             ACCOUNT TRANSACTIONS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return4_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>RETURN MENU</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../gümüst/icons/reply.png</normaloff>
      <normalon>../../gümüst/icons/icns/reply.png</normalon>../gümüst/icons/reply.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="date_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>71</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>CREATING DATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logins_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>LOGINS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aktivites_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>MONEY ACTIVITES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check4_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>CHECK BALANCE</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logins_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="aktivites_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="check_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
